ansible/config/nginx_ckan

```diff
@@ -32,6 +32,11 @@ server {
         # In emergency comment out line to force caching
         # proxy_ignore_headers X-Accel-Expires Expires Cache-Control;
     }
+    location /ncdr/ {
+        access_log  /var/log/nginx/access.log;
+        rewrite /ncdr/(.*) /$1 break;
+    	proxy_pass http://ec2-52-19-157-201.eu-west-1.compute.amazonaws.com;
```

Comment:
Is there a better domain name for this in case this instance ever gets replaced?

Reply:
Suspect it won't ever be a domain name.
Could certainly go in the group vars file and get pushed here as a variable.
Is that now done in sss/main.yml ? If so it'd have to change to `template` syntax in order to pull in the variable?

I do hate the undiffableness of all the not-secret settings in that encrypted file.
This for instance, is a minor implementation detail, not particularly a secret.

Reply:
I don't think the templating helps us much here, it's the only place it's used and should be fairly static 👍